### ISSUE TITLE
Effects on Kwin Don't Work

### ISSUE THREAD

**Issue by ivanv2305** [Schneegans/Burn-My-Windows#376]
## System
KDE Plasma Version: 5.27.7
KDE Frameworks Version: 5.107.0
Qt Version: 5.15.10
Kernel Version: 6.4.0-3-amd64 (64-bit)
Graphics platform: X11

Processors: 6 × Intel® Core™ i5-9400F CPU @ 2.90GHz
Memory: 7,7 GiB RAM
 Graphics processor: NVIDIA GeForce GTX 1050 Ti/PCIe/SSE2
 
 ## Bug Description
 Hello Everyone! I have Some problems.
 I have Debian Sid With KDE 5.27.7 with X11 Session, Using Last Burn-My-Windows (Well, i think that). After installation of Effects pack with wget I can see that Every effect except standart don't work. 
 
 After using this command :
```bash
journalctl -f -o cat | grep -E 'burn-my-windows|'
```
I got this:
```bash
libkwinglutils: Failed to compile fragment shader:
 "0(77) : error C0153: floating point constant overflow\n\x00"
kwin_scripting: /home/kozliv15/.local/share/kwin/effects/kwin4_effect_tv/contents/code/main.js:74: error: Failed to set uniform uColor
libkwinglutils: Failed to compile fragment shader:
 "0(77) : error C0153: floating point constant overflow\n\x00"
kwin_scripting: /home/kozliv15/.local/share/kwin/effects/kwin4_effect_tv_glitch/contents/code/main.js:74: error: Failed to set uniform uDuration
```
When I install effects from KDE Store it also gives me an error (like TV glitch here^^^^)
I tried to reboot, then tried to install libkf5widgetsaddons-dev, and reboot angain, and this doesn't help me...

**Comment by Schneegans** [comment]
Hi there! Thanks a lot for the report! I think I know what could be the reason. Would you like to test this? You need to install the effects from the `fix/#376` branch. Like this:

```
git clone https://github.com/Schneegans/Burn-My-Windows.git
cd Burn-My-Windows
git checkout fix/#376
./kwin/build.sh
mkdir -p ~/.local/share/kwin/effects
tar -xf kwin/burn_my_windows_kwin4.tar.gz -C ~/.local/share/kwin/effects
```

And then re-select the effect in the system settings.

Does it work now?

**Comment by ThisNekoGuy** [comment]
@Schneegans  Can confirm, this fixes it (I'm on Gentoo)
